torch/testing/_internal/common_utils.py

```diff
@@ -789,27 +790,80 @@ def safeCoalesce(self, t):
 
         return tg
 
-    def assertEqual(self, x, y, prec=None, message='', allow_inf=False, exact_dtype=None):
+    # Some analysis of tolerance by logging tests from test_torch.py can be found
+    # in https://github.com/pytorch/pytorch/pull/32538.
+    # dtype name : (rtol, atol)
+    dtype_precisions = {
+        'float16': (0.001, 1e-5),
+        'bfloat16': (0.016, 1e-5),
+        'float32': (1.3e-6, 1e-5),
+        'float64': (1e-7, 1e-7),
+        'complex32': (1.3e-6, 1e-5),
+        'complex64': (1e-7, 1e-7),
```

Comment:
cc @anjali411 here, it seemed like you needed more tolerance for some complex tests, so it's interesting that the empirical numbers here have complex and float line up exactly.

Reply:
talked to Anjali this should have been 64 and 128 to align with the float types, will fix.